test_conformance/compiler/test_compiler_defines_for_extensions.cpp

```diff
@@ -44,15 +45,17 @@ const char *known_extensions[] = {
     "cl_khr_egl_image",
     "cl_khr_egl_event",
     "cl_khr_create_command_queue",
-    "cl_khr_priority_hints",
-    "cl_khr_throttle_hints",
-    "cl_khr_il_program",
```

Comment:
`cl_khr_il_program` is a valid OpenCL 1.2 extension.

Reply:
Good spot, thanks. I've just commited it back in.